layer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM public.ecr.aws/sam/build-python3.7
+FROM public.ecr.aws/lambda/provided:latest
```

Comment:
Does this mean that on every build the version of awscli being used could be different? That seems strange to me. I think it would be better if we could explicitly control the version being used.

Reply:
We can install past versions: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-version.html

However, I need to double check if this is a okay from a security standpoint. We may need to figure out an automated way to upgrade awscli versions -- that was the main motivation behind moving to pip install before I believe (dependabot can catch those upgrades)